src/content/terms-and-policies.md

```diff
@@ -0,0 +1,378 @@
+---
```

Comment:
Is this a standard terms and policies document that you copied over from somewhere? And did you read all of it?

Reply:
I wrote it from scratch!